docs/src/examples/components/Avatar/Variations/AvatarExampleSize.shorthand.tsx

```diff
@@ -5,15 +5,22 @@ import { Avatar } from '@stardust-ui/react'
 const AvatarExampleSizeShorthand = () =>
   _.times(10, i => {
     const size = i + 1
-
     return (
-      <div style={{ background: 'white' }}>
-        <Avatar
-          key={size}
-          size={size}
-          src="/public/images/avatar/small/matt.jpg"
-          status="Available"
-        />
+      <div style={{ background: 'white', marginBottom: '10px' }}>
+        <div style={{ background: 'inherit', width: '10%', float: 'left', marginBottom: '10px' }}>
```

Comment:
It is not intuitive that these inline style background properties are required for the example to work.  We don't want our examples dependent on inline styling.

Let's get this working without the inline styles.  Even if we assume for now that the background color is white, it will be preferable to relying on inline styles.

Reply:
As a user, I'd expect this example to work given the following the simplified example:

```jsx
import _ from 'lodash'
import React from 'react'
import { Avatar } from '@stardust-ui/react'

const AvatarExampleSizeShorthand = () =>
  _.times(10, i => {
    const size = i + 1
    return (
      <div key={size}>
        <Avatar size={size} src="public/images/avatar/small/matt.jpg" status="Available" />
        &emsp;
        <Avatar size={size} name="John Doe" status="Available" />
        &emsp;
        <Avatar size={size} src="public/images/avatar/small/matt.jpg" />
      </div>
    )
  })

export default AvatarExampleSizeShorthand
```